config/samples/featuretracker_v1_features.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: features.opendatahub.io/v1
+kind: FeatureTracker
+metadata:
+  name: default-feature
+  labels:
+    app.kubernetes.io/name: default-feature
+    app.kubernetes.io/instance: default
```

Comment:
i thought 
name is like `featuretracker`
instance will be `default-feature`

Reply:
I fixed for consistency. Although, I think this sample is not meant to be used by end-users.